tdrs-backend/tdpservice/users/urls.py

```diff
@@ -1,11 +1,14 @@
 """Routing for Users."""
```

Comment:
These models are under the `users` app, but the urls can be accessed directly. e.g. `/v1/permissions` and `/v1/roles`